src/odemis/driver/simsem.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, name, role, parent, aperture=100e-6, wd=10e-3, **kwargs):
 
         # pixelSize is the same as MD_PIXEL_SIZE, with scale == 1
         # == smallest size/ between two different ebeam positions
-        pxs = fake_img.metadata[model.MD_PIXEL_SIZE]
+        pxs = fake_img.metadata.get(model.MD_PIXEL_SIZE, (1e-6, 1e-6))
```

Comment:
I will create a separate commit in the actual PR for the changes to simsem driver.

Reply:
I will create a new PR for this